### ISSUE TITLE
Mismetching state and country data for US

### ISSUE THREAD

**Issue by arvind-adita** [harpreetkhalsagtbit/country-state-city#34]
There is mismatch in data for US. Example, for Ohio State, there is ohio as a city.

![image](https://user-images.githubusercontent.com/62008502/77724532-37bf5200-7019-11ea-8d59-b33ca43ef3ae.png)

**Comment by harpreetkhalsagtbit** [comment]
1. Ohio city removed
2. Byram is not a city
3. cokato is a city
4. District of Columbia is not a state nor a city - its federal district
5. Lowa is a city
6. Medfield is not a city
7. New Jersey is a state
8. Ontario is a city in californina and Oregon
9. Ramey is not a city
10. Sublimity is a city
11. Trimble is a city

This is what I got from google search, Please let me know if it is correct info. Then I will update accordingly.

**Comment by arvind-adita** [comment]
Yes, also there are many US states you can see as a city. This database need to correct.



--

Kind regards,
Arvind Saharkar


Adita Technologies
M (+91) 989 095 9696
website: www.Aditatechnologies.com<http://www.aditatechnologies.com/>

________________________________

**Comment by beaespi** [comment]
new york is a state and new jersey is a city.
